src/app/controllers/user.py

```diff
@@ -42,21 +42,29 @@ def get_all_users():
     mimetype="application/json"
   )
 
-@users.route("/", methods=["POST"])
+@users.route("/create", methods=["POST"])
 @required_fields(["name", "email", "password"])
 @user_exists()
 def insert_user():
     try:
         user = request.get_json()
-        payload = {
-          "name": user['name'],
-          "email": user["email"],
-          "password": set_password(user["password"])
-        }
-        mongo_client.users.insert_one(payload)
-        return {"sucesso": f"User inserido com sucesso"}, 201
-    except Exception:
-        return {"error": f"Erro ao inserir user: Faltando campo obrigatório"}, 400
+        if check_valid_email(user["email"]) and len(user["password"]) >= 8:
```

Comment:
Criar função que valida tamanho máx. 40
```suggestions
def check_valid_name(name):
     if len(name.split(' ')) >=2 and len(name) <= 40:
         return name
     else:
         return {'error': "Nome inválido."}
```

Reply:
A validação está no JsonSchema, em properties na chave "maxLength".